source/MaterialXShaderGen/Shader.cpp

```diff
@@ -72,7 +72,7 @@ void Shader::initialize(ElementPtr element, ShaderGenerator& shadergen, const Sg
                     // otherwise we can't assign user values to it anyway. 
                     // This is the case for BSDF, EDF, VDF, and shader types 
                     // which we never want to publish as editable uniforms.
-                    const string& typeDefault = shadergen.getSyntax()->getTypeDefault(input->type, true);
+                    const string& typeDefault = shadergen.getSyntax()->getDefaultValue(input->type, true);
```

Comment:
Should this still be getTypeDefault vs getDefaultValue ?

Reply:
getTypeDefault was renamed to getDefaultValue for a more consistent naming and to better describe it's a value being returned.